ui: Change Navigator Design according to latest requirements

Abhishek-kumar09 opened the pull request:
**Description**

Figma: https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=1107%3A5973
- [x] Update drawer icons according to latest drafts
- [ ] Change Names and endpoints according to new design: (Needs approval)

Latest Screenshot:
[Screenshot Collapsed](https://user-images.githubusercontent.com/48255244/126866005-a3d2c132-c1da-4996-9788-7e57f0f19683.png)
[Screenshot Not collapsed](https://user-images.githubusercontent.com/48255244/126866006-7492a78f-5ed4-4c45-b016-ef64391301f4.png) 



This PR fixes #

**Notes for Reviewers**


**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

Abhishek-kumar09 commented on ui/components/Navigator.js:
```diff
@@ -319,10 +315,10 @@ const categories = [
+    icon: <img src="/static/img/drawerIcons/lifecycle_mgmt.svg" transform="shrink-4"  style={drawerIconsStyle} />,
     href: "/management",
-    title: "Management",
+    title: "Lifecycle",
```
Just changed the title from `Management` to `Lifecycle`.
Will have to update all the endpoints name from `management` to `lifecycle` or something similar. If the name is approved.

Abhishek-kumar09 replied:
Need approval on this too:
[open image](https://user-images.githubusercontent.com/48255244/126503618-c5624703-1fe5-41ff-9032-cf5139b1babe.png)

leecalcote replied:
The screenshot here is outdated, right? We still have the separator between MeshMap (an extension) and the other menu items, right?

Abhishek-kumar09 replied:
No I will put that in. Actually, I was signing with none provider and didn't got meshmap there. Hopefully it will solved now.

leecalcote commented:
✔️ The updated icons are looking good.
✔️ The updated menu names are looking good.
Changing the names of the endpoints - - that's a separate concern, right?

Abhishek-kumar09 commented:
Modified pattern of navigation: (Lifecycle management First)
Modified Helper icon design with minimal animation
[Screenshot Collapsed](https://user-images.githubusercontent.com/48255244/126866005-a3d2c132-c1da-4996-9788-7e57f0f19683.png)
[Screenshot Not collapsed](https://user-images.githubusercontent.com/48255244/126866006-7492a78f-5ed4-4c45-b016-ef64391301f4.png)

> Changing the names of the endpoints - - that's a separate concern, right?

Yes, I think it should be done in seperate PR, For now the name is changed from `management` to `Lifecycle`, but I can revert the name and do remaining things in seperate PR.

What do you say? @leecalcote

Ayush-Khandelwal-007 commented:
Hey @Abhishek-kumar09 I think the order the navigation links have changed, do we have a specific reason for that?

Abhishek-kumar09 commented:
> Hey @Abhishek-kumar09 I think the order the navigation links have changed, do we have a specific reason for that?

Yes, Since Meshery is more focused on managing services and networks, The lifecycle  management thing has got the bump.
Actually, It was discussed with @leecalcote in a meeting.

leecalcote commented:
@Abhishek-kumar09 we're almost there. I made a few tweaks to the icons and their arrangement.

On the hep button, will you please:
1. make the help button as large as this in the nav bar (in the upper right hand corner of the screen)
1. Offset the help button from the chevron by:
   1. keeping them as block elements with respect to one another, so that the button not on the same line as the chevron
   1. placing the help button on the left side of the drawer (away from the chevron)

On the Performance and the Profiles menu items, will you ensure that these icons are vertically centered with respect to the "Performance" and "Profiles" text?

Abhishek-kumar09 commented:
- [ ] make the help button as large as this in the nav bar (in the upper right hand corner of the screen)
- [x] Offset the help button from the chevron by:

    - [x] keeping them as block elements with respect to one another, so that the button not on the same line as the chevron
    - [x] placing the help button on the left side of the drawer (away from the chevron).

- [x] eliminate the padding on the left side of each of the icons (see screenshot).
- [x] have the growth animation start by showing buttons closest to the Help button and expanding outward (not expanding inward toward the help button).
- [x] On the Performance and the Profiles menu items vertically aligned
- [x] A new menu item:
- [x]  Will you ensure that the Conformance menu item is not clickable, but on the "Service Mesh Interface" menu item is clickable?

leecalcote commented:
> Hey @Abhishek-kumar09 I think the order the navigation links have changed, do we have a specific reason for that?

Good eye, though, @Ayush-Khandelwal-007

Abhishek-kumar09 commented:
Since we are using custom svg icons now as image, We can't change the color to blue of the image as we can do such with `fontawesomeicons` or `mui-icons`.

leecalcote commented:
> Since we are using custom svg icons now as image, We can't change the color to blue (for the active selected navigator icon) of the image as we can do such with `fontawesomeicons` or `mui-icons`.

Understood. We'll have have to manipulate the SVG style - no problem.

Ayush-Khandelwal-007 commented:
> Since we are using custom svg icons now as image, We can't change the color to blue (for the active selected navigator icon) of the image as we can do such with `fontawesomeicons` or `mui-icons`.

Hey @Abhishek-kumar09 I think we can change colours for these icons too.
if these help you
Reference Links:- 
[fontawesomeicons](https://stackoverflow.com/questions/14474452/can-i-change-the-color-of-font-awesomes-icon-color)
[mui-icons](https://www.geeksforgeeks.org/how-to-change-the-color-of-icons-using-material-ui-in-reactjs/)

leecalcote commented on ui/components/Navigator.js:
```diff
@@ -222,153 +224,110 @@ const styles = (theme) => ({
     id: "Dashboard",
+    icon: <DashboardIcon style={drawerIconsStyle } />,
     href: "/",
     title: "System Dashboard",
```
```suggestion
    title: "Dashboard",
```

Abhishek-kumar09 commented:
Latest screenshots:
1 | 2 |
--- | --- 
![ss1](https://user-images.githubusercontent.com/48255244/127694253-f95a7ea1-d583-4f1b-bf9d-428a0380f54d.png) | ![ss2](https://user-images.githubusercontent.com/48255244/127694311-1585a8b1-e586-4949-8223-6e39fffb47b8.png) |
